### PR TITLE
Bitglass2

### PR DESCRIPTION
**Pull request by asingh-bitglass**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by asingh-bitglass** [comment]
Not sure how to fix requirements.txt, I do have it submitted. Also, the license issue. Thanks

**Review by mmomin-crest — APPROVED** [review]

**Comment by dvora-crest** [comment]
Release notes added to 1.0.14.md file can be moved to the unreleased.md file. Version-specific release notes file will automatically be created on merge.

File: bitglass.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
-    "license": "Copyright (c) 2021-2022 Bitglass App Inc.",
-    "app_version": "1.0.11",
+    "license": "Copyright (c) 2022 alexeiyur",
+    "app_version": "1.1.0",
     "utctime_updated": "2022-01-07T21:14:11.000000Z",
     "package_name": "phantom_bitglass",
     "main_module": "bitglass_connector.py",
-    "min_phantom_version": "5.1.0",
+    "min_phantom_version": "5.3.0",
     "app_wizard_version": "1.0.0",
```
Comment (dvora-crest) [inline bitglass.json:19]: fips_compliant flag is missing

File: LICENSE [hunk 2]
```diff
@@ -1,201 +1,9 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+The MIT License (MIT)
```
Comment (ishans-crest) [inline LICENSE:1]: @mattsayar-splunk 
Contributor has updated the License to MIT and also update permission verbose on line #5, is it okay or we need to revert license back to MIT license

File: bitglass.json (hunk 1 above)
Comment (achandya-crest) [reply bitglass.json:19]: Done

**Review by dvora-crest — APPROVED** [review]

File: LICENSE (hunk 2 above)
Comment (ishans-crest) [reply LICENSE:1]: Hi @asingh-bitglass 

Can you please help us to understand why the License has been changed from Apache License to MIT ?

**Review by ricardo-phantom — APPROVED** [review]

File: LICENSE (hunk 2 above)
Comment (achandya-crest) [reply LICENSE:1]: Keeping MIT license as per discussion with Matt

**Pull request merged**